code/api/src/modules/user/mutations.js

```diff
@@ -27,6 +27,24 @@ export const userSignup = {
   resolve: create
 }
 
+
+// Update
+export const userUpdate = {
```

Comment:
This is the mutation associated with the update resolver. The Graphql mutation will take id and styleResult as args, returns a user back.